src/physics/src/physics_factory.C

```diff
@@ -110,22 +114,22 @@ namespace GRINS
     libmesh_error();
   }
 
-  void visc_error( const std::string& physics,				  
+  void visc_error( const std::string& physics,
```

Comment:
"Fix whitespace while we're in the file anyway" diffs, on the other hand, are fine.